test/test_persist.py

```diff
@@ -44,6 +45,86 @@ def expected_df_2():
         }
     ).to_pandas()
 
+
+@pytest.fixture
+def expected_df_3():
+    return pd.DataFrame(
```

Comment:
Update tests to demo some concrete behavior. Given this schema in place, it's hard to understand what's tested here.